### DISCUSSION TITLE
Hotfix for 5.10.1

### DISCUSSION THREAD

**Discussion by arturcic** [GitTools/GitVersion#3086] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/GitTools/GitVersion/releases/tag/5.10.1'>Hotfix for 5.10.1</a>.</em>

**Comment by cronventis** [comment]
The Title of the release is wrong (5.10.0) and should be 5.10.1
https://github.com/GitTools/GitVersion/releases/tag/5.10.1

**Reply by arturcic** [reply]
> The Title of the release is wrong (5.10.0) and should be 5.10.1 https://github.com/GitTools/GitVersion/releases/tag/5.10.1

Fixed
